inc/interest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Handles Interest functionality.
+ *
+ * @package Pantheon/EdgeIntegrations
+ */
+
+namespace Pantheon\EI\WP\Interest;
+
+/**
+ * Bootstrap Interest functionality.
+ */
+function bootstrap() {
+	$n = function( $function ) {
```

Comment:
I _might_ change this parameter to something else so this is a bit more obvious. Possibly add a comment above, as well, to help our future selves and potential new developers not used to seeing this pattern. I still like it quite a lot, but it's a bit opaque.

```suggestion
	// Helper variable function that simplifies callbacks.
	$n = function( $callback ) {
```

Reply:
Good call, added comments in [10b5c8f](https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations/pull/8/commits/10b5c8faa104b28c30ed65d7b803e707931552cf).